ExtraEnemyCustomization/Utils/Integrations/Inject/Inject_CM_PageRundown_New.cs

```diff
@@ -0,0 +1,27 @@
+﻿using CellMenu;
+using HarmonyLib;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+
+namespace EECustom.Utils.Integrations.Inject
+{
+    [HarmonyPatch(typeof(CM_PageRundown_New), nameof(CM_PageRundown_New.OnEnable))]
+    internal static class Inject_CM_PageRundown_New
+    {
+        public static void Postfix()
+        {
+            object hotreloader = MTFOUtil.HotReloaderField?.GetValue(null) ?? null;
+            if (hotreloader is null)
+                return;
+
+            var buttonField = hotreloader.GetType().GetField("button", BindingFlags.NonPublic | BindingFlags.Instance);
+            if (buttonField is null)
+                return;
+
+            var button = (CM_Item)buttonField.GetValue(hotreloader);
+            button.add_OnBtnPressCallback((Action<int>)MTFOUtil.OnHotReloaded);
```

Comment:
Is there no better way to do this? I have a feeling that we should have implemented it on the MTFO API side.

If not, keep it and we can PR it into MTFO

Reply:
Yeah In indirect way, they only have method that accepts that have interface of `IHotreload`

Reply:
I see, this solution is fine but it might break in the future